### ISSUE TITLE
the "Searching for files" step is unreasonably slow

### ISSUE THREAD

**Issue by thegoodhen** [thegoodhen/AprilTools#8]
Fix it.

**Comment by asmigala** [comment]
In fact, why do we have a separate method for sorting filenames at all? Isn't simply sorting them in a lexicographic order enough? While that wouldn't work when the numbers are not zero-padded (such as `file_1.jpg` to `file10.jpg`, I think that's very rare and easily correctable to the correctable with some shell scripting).

Also, I believe there is a bug in the code where filenames such as `file-01_0001.jpg`, `file-01_0002.jpeg` won't be correctly sorted. That would also be fixed with lexicographic sort.

**Comment by thegoodhen** [comment]
You are correct; I was trying to catch the edge case you mentioned, while creating a bug for the other edge case. I believe that the correct approach would be to parse the number from the filename from the end to the start, possibly ending when the number is too large (6 digits?). This approach would cover both edge cases mentioned while still working for other cases.

**Comment by asmigala** [comment]
The simplest solution to me is simply assuming zero-padded frame numbers, I believe that's what most video software does.

**Comment by thegoodhen** [comment]
You are probably right.  I am about to leave home soon, but I'll reimplement the sorting shortly.
